src/components/ContactPanel/style.css

```diff
@@ -0,0 +1,3 @@
+.header-spacing {
+    margin-bottom: 16px;
+}
```

Comment:
If this class is being used in multiple places, can you define it once in a global stylesheet and import it wherever it's being used?
https://developer.mozilla.org/en-US/docs/Web/CSS/@import